src/timeseriesinsights/azext_timeseriesinsights/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": false,
```

Comment:
Is it better to remove "azext.isPreview": false, as you just experimental?

Reply:
✔